### ISSUE TITLE
Single digit version not recognised

### ISSUE THREAD

**Issue by CptMcSplody** [loot/loot#1159]
For mod [Unique Landscapes - Beaches of Cyrodiil - Lost Coast](https://www.nexusmods.com/oblivion/mods/14720), the version in the description is listed as `version: 2`.

![image](https://user-images.githubusercontent.com/1944639/67105479-c2756880-f1c0-11e9-86da-ef680b099620.png)

![image](https://user-images.githubusercontent.com/1944639/67105664-16804d00-f1c1-11e9-9275-aa69c668f4b8.png)

I noticed it when trying to replace this message with a version check and alias.

```yaml
  - name: 'xulBeachesOfCyrodiilLostCoast.esp'
    msg:
      - type: say
        content: 'Make sure version is 1.6.3 or later.'

  - name: 'xulBeachesOfCyrodiilLostCoast.esp'
    msg:
      - <<: *obsolete
        condition: 'version("xulBeachesOfCyrodiilLostCoast.esp", "2", <)'
```

**Comment by Ortham** [comment]
This is now fixed in libloot, I'll keep this issue open until LOOT has been updated to use whatever release ends up including the fix.

**Comment by CptMcSplody** [comment]
Okay, thanks.

**Comment by Ortham** [comment]
Closing this as the fix was merged in 0c6b0eb70054fe48051e54a09792c46e3abde4a4.
